lib/process-services-cloud/src/lib/task/start-task/components/people-cloud/people-cloud.component.ts

```diff
@@ -56,6 +56,10 @@ export class PeopleCloudComponent implements OnInit, OnChanges {
     @Input()
     roles: string[];
 
+    /** preselect values validation flag */
```

Comment:
Can we add a little bit of context? The documentation should be clear for a user that doesn't know anything about the component. 
This flag enables the validation on the preSelectUsers passed as input. In case the flag is true the components call the identity service to verify the validity of the information passed as input. Otherwise, no check will be done.